app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Api 
+    module V1
+        class OrdersController < ApplicationController
+            def index
```

Comment:
please use 2 spaces for indentation and leave a new line between method declaration - can you update this everywhere?

Reply:
Please take a look now